src/selectors.js

```diff
@@ -0,0 +1,11 @@
+import * as NotesMindMapSelectors from './components/NotesMindMap/NotesMindMapSelectors';
+import * as UploadsSelectors from './components/Uploads/UploadsSelectors';
+
+/**
+ * Global selectors
+ *
+ * Note!!!
+ * Don't import it to local component selectors
```

Comment:
Where should we import them?
Why do we need them?
Looks quite weird.
Can we refactor to get rid of global selectors?
Which problem requires global selectors?

Reply:
>> Can we refactor to get rid of global selectors?

Yes, we can, but anyway we need to follow some practice to make selectors from different modules available to use each other without a circular dependency issue and with control how selectors are using each other.

>> Why do we need them?

To avoid circular dependencies when different selectors uses each other and to control the way how different selectors uses each other

>> Which problem requires global selectors?

https://randycoulman.com/blog/2016/09/27/modular-reducers-and-selectors/
https://randycoulman.com/blog/2018/06/12/solving-circular-dependencies-in-modular-redux/

Reply:
Okay. Please add to the comment where should we import these selectors. The comment says where not to use them, but doesn't explain where is it ok to use these global selectors. Where should we import them? In other global selectors? In `NotesMindMapContainer`? By "local selectors" do you mean selectors declared in `NotesMindMapComponent` or which ones?
You can link to the articles in the comment as well.
Otherwise it's not clear where to use these selectors (for me :) ).

Reply:
changed the comment